src/Moka/Plugin/PHPUnit/PHPUnitPlugin.php

```diff
@@ -6,11 +6,19 @@
 use Moka\Exception\PluginNotRegisteredException;
 use Moka\Moka;
 use Moka\Plugin\PluginInterface;
+use Moka\Strategy\MockingStrategyInterface;
```

Comment:
The circular dependency is still there as plugins require interfaces defined in Moka.